### ISSUE TITLE
rpy2 version on conda outdated since Jan 2020

### ISSUE THREAD

**Issue by wtsf** [conda-forge/rpy2-feedstock#62]
Issue:
The latest version of rpy2 (2.9.4) available on conda forge is not compatible with pandas > 1.0.

It breaks when trying to import a pandas dataframe, relying on the now invalid:
`PandasDataFrame.from_items(items)`

So as soon as you issue the import (here in a Jupyter Notebook):
`df = pd.DataFrame(list('1234'))`
`%%R -i df`

You get the Attribute error:
`AttributeError: type object 'DataFrame' has no attribute 'from_items'`

This is detailed at the rpy2 site, and has been fixed in rpy2 > 3.3:
https://github.com/rpy2/rpy2/issues/680

It seems that conda-forge is falling behind with rpy2... forcing people back to pip with all its problems
<br/>
Environment (<code>conda list</code>):

<details>

```
# packages in environment at /opt/anaconda3/envs/RwPython:
#
# Name                    Version                   Build  Channel
_r-mutex                  1.0.0               anacondar_1  
appnope                   0.1.0                    py37_0  
attrs                     19.3.0                     py_0  
backcall                  0.2.0                      py_0  
blas                      1.0                         mkl  
bleach                    3.1.5                      py_0  
bwidget                   1.9.11                        1  
bzip2                     1.0.8                h1de35cc_0  
ca-certificates           2020.6.24                     0  
cairo                     1.14.12              hc4e6be7_4  
cctools                   927.0.2              h5ba7a2e_4  
certifi                   2020.6.20                py37_0  
clang                     10.0.0          default_hf57f61e_0  
clang_osx-64              10.0.0               h05bbb7f_1  
clangxx                   10.0.0          default_hf57f61e_0  
clangxx_osx-64            10.0.0               h05bbb7f_1  
compiler-rt               10.0.0               h47ead80_0  
compiler-rt_osx-64        10.0.0               hbcc88fd_0  
curl                      7.67.0               ha441bb4_0  
decorator                 4.4.2                      py_0  
defusedxml                0.6.0                      py_0  
entrypoints               0.3                      py37_0  
fontconfig                2.13.0               h5d5b041_1  
freetype                  2.10.2               ha233b18_0  
fribidi                   1.0.9                haf1e3a3_0  
gettext                   0.19.8.1             h15daf44_3  
gfortran_osx-64           4.8.5                h22b1bf0_8  
glib                      2.63.1               hd977a24_0  
graphite2                 1.3.14               h38d11af_0  
gsl                       2.4                  h1de35cc_4  
harfbuzz                  2.4.0                h831d699_0  
icu                       58.2                 h0a44026_3  
importlib-metadata        1.7.0                    py37_0  
importlib_metadata        1.7.0                         0  
intel-openmp              2019.4                      233  
ipykernel                 5.3.4            py37h5ca1d4c_0  
ipython                   7.16.1           py37h5ca1d4c_0  
ipython_genutils          0.2.0                    py37_0  
jedi                      0.17.1                   py37_0  
jinja2                    2.11.2                     py_0  
jpeg                      9b                   he5867d9_2  
jsonschema                3.2.0                    py37_0  
jupyter_client            6.1.6                      py_0  
jupyter_core              4.6.3                    py37_0  
krb5                      1.16.4               hddcf347_0  
ld64                      450.3                h3c32e8a_4  
libcurl                   7.67.0               h051b688_0  
libcxx                    10.0.0                        1  
libedit                   3.1.20191231         h1de35cc_1  
libffi                    3.2.1                h0a44026_6  
libgfortran               3.0.1                h93005f0_2  
libiconv                  1.16                 h1de35cc_0  
libllvm10                 10.0.0               h21ff451_2  
libpng                    1.6.37               ha441bb4_0  
libsodium                 1.0.18               h1de35cc_0  
libssh2                   1.9.0                ha12b0ac_1  
libtiff                   4.1.0                hcb84e12_1  
libxml2                   2.9.10               h3b9e6c8_1  
llvm-openmp               10.0.0               h28b9765_0  
llvm-tools                10.0.0               h21ff451_2  
lz4-c                     1.9.2                hb1e8313_1  
make                      4.2.1                h3efe00b_1  
markupsafe                1.1.1            py37h1de35cc_0  
mistune                   0.8.4            py37h1de35cc_0  
mkl                       2019.4                      233  
mkl-service               2.3.0            py37hfbe908c_0  
mkl_fft                   1.1.0            py37hc64f4ea_0  
mkl_random                1.1.1            py37h959d312_0  
nbconvert                 5.6.1                    py37_0  
nbformat                  5.0.7                      py_0  
ncurses                   6.2                  h0a44026_1  
notebook                  6.0.3                    py37_0  
numpy                     1.19.1           py37h3b9f5b6_0  
numpy-base                1.19.1           py37hcfb5961_0  
openssl                   1.1.1g               h1de35cc_0  
packaging                 20.4                       py_0  
pandas                    1.1.0            py37hb1e8313_0  
pandoc                    2.10.1                        0  
pandocfilters             1.4.2                    py37_1  
pango                     1.42.4               hc249df1_1  
parso                     0.7.0                      py_0  
pcre                      8.44                 hb1e8313_0  
pexpect                   4.8.0                    py37_0  
pickleshare               0.7.5                    py37_0  
pip                       20.2.2                   py37_0  
pixman                    0.40.0               haf1e3a3_0  
prometheus_client         0.8.0                      py_0  
prompt-toolkit            3.0.5                      py_0  
ptyprocess                0.6.0                    py37_0  
pygments                  2.6.1                      py_0  
pyparsing                 2.4.7                      py_0  
pyrsistent                0.16.0           py37h1de35cc_0  
python                    3.7.7           hfe9666f_0_cpython  
python-dateutil           2.8.1                      py_0  
pytz                      2020.1                     py_0  
pyzmq                     19.0.1           py37hb1e8313_1  
r-askpass                 1.0               r36h1de35cc_0  
r-assertthat              0.2.1             r36h6115d3f_0  
r-backports               1.1.4             r36h46e59ec_0  
r-base                    3.6.1                hcb44179_1  
r-base64enc               0.1_3             r36h46e59ec_4  
r-bh                      1.69.0_1          r36h6115d3f_0  
r-bit                     1.1_14            r36h46e59ec_0  
r-bit64                   0.9_7             r36h46e59ec_0  
r-blob                    1.1.1             r36h6115d3f_0  
r-boot                    1.3_20            r36h6115d3f_0  
r-broom                   0.5.2             r36h6115d3f_0  
r-callr                   3.2.0             r36h6115d3f_0  
r-caret                   6.0_83            r36h46e59ec_0  
r-cellranger              1.1.0             r36h6115d3f_0  
r-class                   7.3_15            r36h46e59ec_0  
r-cli                     1.1.0             r36h6115d3f_0  
r-clipr                   0.6.0             r36h6115d3f_0  
r-cluster                 2.0.8             r36hfffe0aa_0  
r-codetools               0.2_16            r36h6115d3f_0  
r-colorspace              1.4_1             r36h46e59ec_0  
r-crayon                  1.3.4             r36h6115d3f_0  
r-curl                    3.3               r36h46e59ec_0  
r-data.table              1.12.2            r36h46e59ec_0  
r-dbi                     1.0.0             r36h6115d3f_0  
r-dbplyr                  1.4.0             r36h6115d3f_0  
r-dichromat               2.0_0             r36h6115d3f_4  
r-digest                  0.6.18            r36h46e59ec_0  
r-dplyr                   0.8.0.1           r36h466af19_0  
r-ellipsis                0.1.0             r36h46e59ec_0  
r-essentials              3.6.0                     r36_0  
r-evaluate                0.13              r36h6115d3f_0  
r-fansi                   0.4.0             r36h46e59ec_0  
r-forcats                 0.4.0             r36h6115d3f_0  
r-foreach                 1.4.4             r36h6115d3f_0  
r-foreign                 0.8_71            r36h46e59ec_0  
r-formatr                 1.6               r36h6115d3f_0  
r-fs                      1.2.7             r36h466af19_0  
r-generics                0.0.2             r36h6115d3f_0  
r-ggplot2                 3.1.1             r36h6115d3f_0  
r-glmnet                  2.0_16            r36hfffe0aa_0  
r-glue                    1.3.1             r36h46e59ec_0  
r-gower                   0.2.0             r36h46e59ec_0  
r-gtable                  0.3.0             r36h6115d3f_0  
r-haven                   2.1.0             r36h466af19_0  
r-hexbin                  1.27.2            r36hfffe0aa_0  
r-highr                   0.8               r36h6115d3f_0  
r-hms                     0.4.2             r36h6115d3f_0  
r-htmltools               0.3.6             r36h466af19_0  
r-htmlwidgets             1.3               r36h6115d3f_0  
r-httpuv                  1.5.1             r36h466af19_0  
r-httr                    1.4.0             r36h6115d3f_0  
r-ipred                   0.9_8             r36h46e59ec_0  
r-irdisplay               0.7.0             r36h6115d3f_0  
r-irkernel                0.8.15                    r36_0  
r-iterators               1.0.10            r36h6115d3f_0  
r-jsonlite                1.6               r36h46e59ec_0  
r-kernsmooth              2.23_15           r36hfffe0aa_4  
r-knitr                   1.22              r36h6115d3f_0  
r-labeling                0.3               r36h6115d3f_4  
r-later                   0.8.0             r36h466af19_0  
r-lattice                 0.20_38           r36h46e59ec_0  
r-lava                    1.6.5             r36h6115d3f_0  
r-lazyeval                0.2.2             r36h46e59ec_0  
r-lubridate               1.7.4             r36h466af19_0  
r-magrittr                1.5               r36h6115d3f_4  
r-maps                    3.3.0             r36h46e59ec_0  
r-markdown                0.9               r36h46e59ec_0  
r-mass                    7.3_51.3          r36h46e59ec_0  
r-matrix                  1.2_17            r36h46e59ec_0  
r-memoise                 1.1.0             r36h6115d3f_0  
r-mgcv                    1.8_28            r36h46e59ec_0  
r-mime                    0.6               r36h46e59ec_0  
r-modelmetrics            1.2.2             r36h466af19_0  
r-modelr                  0.1.4             r36h6115d3f_0  
r-munsell                 0.5.0             r36h6115d3f_0  
r-nlme                    3.1_139           r36hfffe0aa_0  
r-nnet                    7.3_12            r36h46e59ec_0  
r-numderiv                2016.8_1          r36h6115d3f_0  
r-openssl                 1.3               r36h46e59ec_0  
r-pbdzmq                  0.3_3             r36hcccc9c6_1  
r-pillar                  1.3.1             r36h6115d3f_0  
r-pkgconfig               2.0.2             r36h6115d3f_0  
r-plogr                   0.2.0             r36h6115d3f_0  
r-plyr                    1.8.4             r36h466af19_0  
r-prettyunits             1.0.2             r36h6115d3f_0  
r-processx                3.3.0             r36h46e59ec_0  
r-prodlim                 2018.04.18        r36h466af19_0  
r-progress                1.2.0             r36h6115d3f_0  
r-promises                1.0.1             r36h466af19_0  
r-ps                      1.3.0             r36h46e59ec_0  
r-purrr                   0.3.2             r36h46e59ec_0  
r-quantmod                0.4_14            r36h6115d3f_0  
r-r6                      2.4.0             r36h6115d3f_0  
r-randomforest            4.6_14            r36hfffe0aa_0  
r-rbokeh                  0.6.3                     r36_0  
r-rcolorbrewer            1.1_2             r36h6115d3f_0  
r-rcpp                    1.0.1             r36h466af19_0  
r-rcpproll                0.3.0             r36h466af19_0  
r-readr                   1.3.1             r36h466af19_0  
r-readxl                  1.3.1             r36h466af19_0  
r-recipes                 0.1.5             r36h6115d3f_0  
r-recommended             3.6.0                     r36_0  
r-rematch                 1.0.1             r36h6115d3f_0  
r-repr                    0.19.2            r36h6115d3f_0  
r-reprex                  0.2.1             r36h6115d3f_0  
r-reshape2                1.4.3             r36h466af19_0  
r-rlang                   0.3.4             r36h46e59ec_0  
r-rmarkdown               1.12              r36h6115d3f_0  
r-rpart                   4.1_15            r36h46e59ec_0  
r-rsqlite                 2.1.1             r36h466af19_0  
r-rstudioapi              0.10              r36h6115d3f_0  
r-rvest                   0.3.3             r36h6115d3f_0  
r-scales                  1.0.0             r36h466af19_0  
r-selectr                 0.4_1             r36h6115d3f_0  
r-shiny                   1.3.2             r36h6115d3f_0  
r-sourcetools             0.1.7             r36h466af19_0  
r-spatial                 7.3_11            r36h46e59ec_4  
r-squarem                 2017.10_1         r36h6115d3f_0  
r-stringi                 1.4.3             r36h466af19_0  
r-stringr                 1.4.0             r36h6115d3f_0  
r-survival                2.44_1.1          r36h46e59ec_0  
r-sys                     3.2               r36h46e59ec_0  
r-tibble                  2.1.1             r36h46e59ec_0  
r-tidyr                   0.8.3             r36h466af19_0  
r-tidyselect              0.2.5             r36h466af19_0  
r-tidyverse               1.2.1             r36h6115d3f_0  
r-timedate                3043.102          r36h6115d3f_0  
r-tinytex                 0.12              r36h6115d3f_0  
r-ttr                     0.23_4            r36hfffe0aa_0  
r-utf8                    1.1.4             r36h46e59ec_0  
r-uuid                    0.1_2             r36h46e59ec_4  
r-viridislite             0.3.0             r36h6115d3f_0  
r-whisker                 0.3_2             r36h6115d3f_4  
r-withr                   2.1.2             r36h6115d3f_0  
r-xfun                    0.6               r36h6115d3f_0  
r-xml2                    1.2.0             r36h466af19_0  
r-xtable                  1.8_4             r36h6115d3f_0  
r-xts                     0.11_2            r36h46e59ec_0  
r-yaml                    2.2.0             r36h46e59ec_0  
r-zoo                     1.8_6             r36h46e59ec_0  
readline                  7.0                  h1de35cc_5  
rpy2                      2.9.4           py37r36h1d22016_0    r
send2trash                1.5.0                    py37_0  
setuptools                49.6.0                   py37_0  
simplegeneric             0.8.1                    py37_2  
six                       1.15.0                     py_0  
sqlite                    3.32.3               hffcf06c_0  
tapi                      1000.10.8            ha1b3eb9_0  
terminado                 0.8.3                    py37_0  
testpath                  0.4.4                      py_0  
tk                        8.6.10               hb0a8c7a_0  
tktable                   2.10                 h1de35cc_0  
tornado                   6.0.4            py37h1de35cc_1  
traitlets                 4.3.3                    py37_0  
tzlocal                   2.1                      py37_0  
wcwidth                   0.2.5                      py_0  
webencodings              0.5.1                    py37_1  
wheel                     0.34.2                   py37_0  
xz                        5.2.5                h1de35cc_0  
zeromq                    4.3.2                hb1e8313_2  
zipp                      3.1.0                      py_0  
zlib                      1.2.11               h1de35cc_3  
zstd                      1.4.5                h41d2c2f_0  

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : RwPython
    active env location : /opt/anaconda3/envs/RwPython
            shell level : 1
       user config file : /Users/wtsf/.condarc
 populated config files : /Users/wtsf/.condarc
          conda version : 4.8.3
    conda-build version : 3.18.11
         python version : 3.8.3.final.0
       virtual packages : __osx=10.12.6
       base environment : /opt/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /opt/anaconda3/pkgs
                          /Users/wtsf/.conda/pkgs
       envs directories : /opt/anaconda3/envs
                          /Users/wtsf/.conda/envs
               platform : osx-64
             user-agent : conda/4.8.3 requests/2.24.0 CPython/3.8.3 Darwin/16.7.0 OSX/10.12.6
                UID:GID : 502:20
             netrc file : None
           offline mode : False

```
</details>

**Comment by ocefpaf** [comment]
r2py is a beast to build and with multiple patches that are hard to maintain. If you want to take a swing at it please take a look on the existing PRs and see if you can help.

**Comment by wtsf** [comment]
Thanks but that’s over my pay grade - I’ll just retreat to older versions.
T

> On 19 Aug 2020, at 13:42, Filipe <notifications@github.com> wrote:
> 
> r2py is a beast to build and with multiple patches that are hard to maintain. If you want to take a swing at it please take a look on the existing PRs and see if you can help.
> 
> 

**  Prof. W. Tecumseh Fitch, PhD                    http://becogbio.univie.ac.at/   **
**  Dept. of Behavioral and Cognitive Biology, University of Vienna, Austria  **
**  +43-1-4277-76111 office;               email:  tecumseh.fitch@univie.ac.at    **

**Comment by ocefpaf** [comment]
> Thanks but that’s over my pay grade

Also mine :smile: 

But we can ask for help outside of the community here since we cannot handle this at the moment.

**Comment by matthewwardrop** [comment]
This might be closed by #66 .

**Comment by wtsf** [comment]
Thanks, I’ll give it a try! Tecumseh

> On 14 Oct 2020, at 06:27, Matthew Wardrop <notifications@github.com> wrote:
> 
> This might be closed by #66 .

**  Prof. W. Tecumseh Fitch, PhD                    http://becogbio.univie.ac.at/   **
**  Dept. of Behavioral and Cognitive Biology, University of Vienna, Austria  **
**  +43-1-4277-76111 office;               email:  tecumseh.fitch@univie.ac.at    **

**Comment by dbast** [comment]
Can this be closed?

rpy2 3.3.6 (current upstream version) got available via https://github.com/conda-forge/rpy2-feedstock/pull/70
